themes/finna2/templates/RecordDriver/DefaultRecord/container-links.phtml

```diff
@@ -1,23 +1,43 @@
 <!-- START of: finna - RecordDriver/DefaultRecord/container-links.phtml -->
+<?php $this->inRecordInfo ?? false; ?>
```

Comment:
Mikä tämä on? Ei tee käytännössä mitään. Jos tarkoitus oli varmistaa, että `$this->inRecordInfo` on asetettu, niin ei tarvitse. Se palauttaa oletuksena null.